core/services/feeds/models.go

```diff
@@ -0,0 +1,87 @@
+package feeds
+
+import (
+	"database/sql/driver"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/lib/pq"
+)
+
+type ManagerService struct {
+	ID        int32
+	Name      string
+	URI       string
+	PublicKey PublicKey
+	JobTypes  pq.StringArray
+	Network   string
+	CreatedAt time.Time
+	UpdatedAt time.Time
+}
+
+func (ManagerService) TableName() string {
```

Comment:
I did it this way because the package name was already feeds, so it didn't need to be `feeds.FeedsManager` but I can also see that as being inconsistent with our naming convention. I'll make the change